rosdep/python.yaml

```
@@ -10142,6 +10142,16 @@ tilestache:
   debian: [tilestache]
   fedora: [python-tilestache]
   ubuntu: [tilestache]
+tinkerforge:
```

```suggestion
tinkerforge-pip:
```

Can we add the `-pip`, this helps easily identify the dependency as a pip package and reduces the chances of a future clash of the label with a system package.

---

Actually, to pass our naming this needs to be `python3-tinkerforge-pip` (and needs to be moved into the correct place in the file to be alphabetical).